packages/vuetify/src/components/VProgressCircular/VProgressCircular.tsx

```diff
@@ -0,0 +1,142 @@
+// Styles
+import './VProgressCircular.sass'
+
+// Composables
+import { makeTagProps } from '@/composables/tag'
+import { useTheme } from '@/composables/theme'
+import { useColor } from '@/composables/color'
+
+// Utilities
+import { computed, defineComponent } from 'vue'
+import { convertToUnit, makeProps } from '@/util'
+
+export default defineComponent({
+  name: 'VProgressCircular',
+
+  props: makeProps({
+    indeterminate: Boolean,
+    rotate: {
+      type: [Number, String],
+      default: 0,
+    },
+    size: {
```

Comment:
use size composable

Reply:
This is looking for a number that it uses to inform the svg viewbox height and width. Since the size composable returns strings, I'm not sure how I would get the to work...